openedx/features/learner_analytics/views.py

```diff
@@ -79,7 +79,7 @@ def get(self, request, course_id):
         if (has_access):
             grading_policy = course.grading_policy
 
-            (grade_data, answered_percent) = self.get_grade_data(request.user, course_key, grading_policy['GRADE_CUTOFFS'])
+            (grade_data, answered_percent, passing_grade, percent_grade) = self.get_grade_data(request.user, course_key, grading_policy['GRADE_CUTOFFS'])
```

Comment:
FYI: When it get to more than 2 values, usually switch to a dict, but fine for test code.